# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1363319636 14400
#      Thu Mar 14 23:53:56 2013 -0400
# Node ID e7f6c479ce1ccbe977e4040600ea91186e14bc3b
# Parent  e608de76742cf0f0d96729364bf8f61a94b4b331
* __xzip__.m: Use parens around anonymous function expression that spans multiple lines.

diff --git a/scripts/miscellaneous/private/__xzip__.m b/scripts/miscellaneous/private/__xzip__.m
--- a/scripts/miscellaneous/private/__xzip__.m
+++ b/scripts/miscellaneous/private/__xzip__.m
@@ -57,19 +57,20 @@ function entries = __xzip__ (commandname
     mkdir (outdir);
   endif
 
   cwd = pwd ();
   unwind_protect
     files = glob (files);
 
     ## Ignore any file with the compress extension
-    files(cellfun (@(x) length (x) > length (extension)
-      && strcmp (x((end - length (extension) + 1):end), extension),
-      files)) = [];
+    files(cellfun (@(x) (length (x) > length (extension)
+                         && strcmp (x((end - length (extension) + 1):end),
+                                    extension)),
+                   files)) = [];
 
     copyfile (files, outdir);
 
     [d, f] = myfileparts (files);
 
     cd (outdir);
 
     cmd = sprintf (commandtemplate, sprintf (" %s", f{:}));
